scripts/local-bazel-build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
```

Comment:
Feel free to remove this one and or replace it with your employer in the followup PR

Reply:
Fixed!